src/addrman.cpp

```diff
@@ -277,14 +277,14 @@ bool CAddrMan::Add_(const CAddress& addr, const CNetAddr& source, int64_t nTimeP
 
         // stochastic test: previous nRefCount == N: 2^N times harder to increase it
         int nFactor = 1;
-        for (int n = 0; n < pinfo->nRefCount; n++)
+        for (int n = 0; n < pinfo->nRefCount; ++n)
```

Comment:
In the subject you mention "for non-trivial types". I always thought that int was a trivial type.

Reply:
I've explained this in the commit message body:

> All the modified files fix this for some objects of non-trivial type. Increments/decrements of trivial types in these files were also converted to prefix instead of postfix (wherever the result isn't used) for consistency.

But, I'll change the subject to reflect that as well.